### PR TITLE
Medbot centcom announcer and relative station trait.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
Adds a new centcom announcer datum and a relative station trait. Credits to @deathride58 since he originally made this back in late 2019 on citadel; I'm more or less just porting it (sort of, code is not the same).
I also removed a duplicate .ogg found in `sound/ai/intern`

## Why It's Good For The Game
The idea of the announcement system being occasionally replaced by a sbaitso-voiced bot is cool.

EDIT: Samples, as requested by bobbahbrown, for those who can't download sound files I guess:
https://vocaroo.com/1ov3OmRKbQGQ
https://vocaroo.com/15QNT9FGdh92

## Changelog
:cl:
add: There is a new (backup) centcom announcer in town.
/:cl:

### CODE REVIEW

**Comment by bobbahbrown** [comment]
can you provide a sample

**Comment by Ghommie** [comment]
> 
> 
> can you provide a sample

It's basically how you would expect Dr.Sbaitso "medbot" voice to sound like. But ok.

**Comment by bobbahbrown** [comment]
thank you i like your pr its very nice

**Comment by Serxule** [comment]
idea to go with this: all the bots that spawn and that get made look like medbots but with their respective differences, eg: secbot is a medbot with a stun baton, etc.

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -58,7 +58,20 @@
 	weight = 1
 	show_in_report = TRUE
 	report_message = "Please be nice to him."
+	blacklist = list(/datum/station_trait/announcement_intern)
```
Comment (Qustinnus) [inline code/datums/station_traits/neutral_traits.dm:61]:
```suggestion
	blacklist = list(/datum/station_trait/announcement_medbot)
```

Comment (Ghommie) [reply code/datums/station_traits/neutral_traits.dm:61]: Ah, good catch, man.

**Comment by TheBonded** [comment]
> idea to go with this: all the bots that spawn and that get made look like medbots but with their respective differences, eg: secbot is a medbot with a stun baton, etc.

And let you flip them too

**Review by Mothblocks — CHANGES_REQUESTED** [review]
I'm starting to get concerned with the amount of oggs we have for super niche features like this. What's the total file size?

File: code/datums/announcers/medbot_announcer.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+						ANNOUNCER_METEORS = 'sound/ai/medbot/meteors.ogg',
+						ANNOUNCER_OUTBREAK5 = 'sound/ai/medbot/outbreak5.ogg',
+						ANNOUNCER_OUTBREAK7 = 'sound/ai/medbot/outbreak7.ogg',
+						ANNOUNCER_POWEROFF = 'sound/ai/medbot/poweroff.ogg',
+						ANNOUNCER_POWERON = 'sound/ai/medbot/poweron.ogg',
+						ANNOUNCER_RADIATION = 'sound/ai/medbot/radiation.ogg',
+						ANNOUNCER_SHUTTLECALLED = 'sound/ai/medbot/shuttlecalled.ogg',
+						ANNOUNCER_SHUTTLEDOCK = 'sound/ai/medbot/shuttledock.ogg',
+						ANNOUNCER_SHUTTLERECALLED = 'sound/ai/medbot/shuttlerecalled.ogg',
+						ANNOUNCER_SPANOMALIES = 'sound/ai/medbot/spanomalies.ogg')
```
[the remark this answers is not in the document]
Comment (Ghommie) [reply code/datums/announcers/medbot_announcer.dm]: Ok, I hope I have done exactly what you told me to do here.

**Comment by Ghommie** [comment]
> 
> 
> I'm starting to get concerned with the amount of oggs we have for super niche features like this. What's the total file size?

1.88 MBs on my disk, almost twice the default (1.01 MBs) and roughly a third of the intern (5.77 MBs). Do you want me to (try to) compress it?

**Comment by Qustinnus** [comment]
if filesize is a concern remove shit like aurora caelus and the detective ambiences which are literaly just super long music pieces that noone ever hears.

File: code/datums/announcers/medbot_announcer.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (Mothblocks) [reply code/datums/announcers/medbot_announcer.dm]: It's good enough.

**Comment by Mothblocks** [comment]
I think we should.

**Pull request merged**